assets/scss/coder-dark.scss

```
@@ -1,10 +1,10 @@
 @import "variables";
 @import "base_dark";
 @import "content_dark";
-@import "notices_dark";
+//@import "notices_dark";
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)